src/views/sessions/SessionList/BaseList.vue

```diff
@@ -137,7 +145,8 @@ export default {
               hasUpdate: false,
               extraActions: this.extraActions
             }
-          }
+          },
+          ...this.columnsMeta
         }
       },
       headerActions: {
```

Comment:
There seem to be an issue with the way the "extraActions" array is being accessed and stored. It should either include or exclude the 'terminal' column based on its presence in columnsExclude instead of accessing it directly.

Additionally, there seems to be a duplication since the "columnsMeta" property appears twice (once in data) but it's not clear how they relate. Perhaps you meant to set "hasUpdate", extraActions etc inside one of these object properties rather than within them?

It would help if you could elaborate what exactly this code does so I have a better understanding of where to start addressing these issues.